tests/Doctrine/Common/Lexer/AbstractLexerTest.php

```diff
@@ -20,15 +20,12 @@ public function setUp() : void
         $this->concreteLexer = new ConcreteLexer();
     }
 
-    /**
-     * {@inheritdoc}
-     */
     public function tearDown() : void
     {
         setlocale(LC_ALL, null);
     }
 
-    public function dataProvider()
+    public function dataProvider() : array
```

Comment:
As these `: array` additions are under `tests/`, should be a BC neither a problem :)